### PR TITLE
Fall back to compiled-in queries for supported languages.

### LINKED ISSUE #542: Make sure the release binaries have working queries out of the box
**Is your feature request related to a problem? Please describe.**
We are now releasing binaries with `cargo-dist`. See #538. But we are not distributing any query files. This means that a downloaded package will not work out of the box.

**Describe the solution you'd like**
Either we have to find a way to include the query files with `cargo-dist`, or, probably preferably, the Topiary CLI can use the new convenience functions to access compile-time query files. See #528.

**Describe alternatives you've considered**
The user can always download query files manually.

### PR DESCRIPTION
**Pull request by torhovland**
Fixes #542.

### CODE REVIEW

File: topiary-cli/src/main.rs [hunk 1]
```diff
@@ -123,30 +123,41 @@ async fn run() -> CLIResult<()> {
...
+                "Could not open query file".into(),
+                Some(CLIError::IOError(e)),
+            )
+        })
+    })
+    .and_then(|query_content: String| Ok(TopiaryQuery::new(&grammar, &query_content)?))
+    .or_else(|e| {
+        // If we weren't able to read the query file, and the user didn't
+        // request a specific query file, we should fall back to the built-in
+        // queries.
```
Comment (ErinvanderVeen) [inline topiary-cli/src/main.rs:153]: Topiary should log that it is falling back to the builtin query file to avoid future headaches in debugging.

Comment (torhovland) [reply topiary-cli/src/main.rs:153]: Good point!

Comment (torhovland) [reply topiary-cli/src/main.rs:153]: Check now.

**Comment by torhovland** [comment]
@ErinvanderVeen This changed a bit now, after a rebase on main. Please take another look.

**Review by ErinvanderVeen — APPROVED** [review]

**Pull request merged**